snapcraft.yaml

```diff
@@ -101,3 +101,23 @@ parts:
       craftctl default
       # ensure build-in tests are run
       cd ${CRAFT_PART_SRC} && make test TESTDIR=${CRAFT_PRIME}
+    stage:
```

Comment:
While I like that we can use stage for these things, it's not consistent with how we usually remove stuff from the base. We usually remove stuff in the hooks, and it'll be confusing if things are split. Maybe @alfonsosanchezbeato has an opinion on this